books.js

```diff
@@ -0,0 +1,77 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable camelcase */
+/* eslint-disable max-classes-per-file */
+class Book {
+  id='';
+
+  title='';
+
+  author='';
+
+  constructor(param_id, param_title, param_author) {
+    this.id = param_id;
+    this.title = param_title;
+    this.author = param_author;
+  }
+
+  generateBookLi(index) {
+    return `<li class="book"><span class="book_details">"${this.title
+    }" by ${this.author}</span><button class="remove_button" onClick="removeBook(${index})">Remove</button></li>`;
+  }
+}
+
+class BookPage {
+  booklist = [];
+
+  // eslint-disable-next-line no-useless-constructor, no-empty-function
+  constructor() {
+
+  }
+
+  storeInMemory() {
+    localStorage.setItem('pageBooks', JSON.stringify(this.booklist));
+  }
+
+  remove(index) {
+    this.booklist.splice(index, 1);
+    this.storeInMemory();
+  }
+
+  addBook(book) {
+    this.booklist.push(book);
```

Comment:
- [ ] [OPTIONAL] It would be better if you use the higher order functions like `concat` as they do not mutate the array rather they return a copy of the original array.